airflow/www/extensions/init_views.py

```diff
@@ -0,0 +1,118 @@
+    appbuilder.add_view(views.LogModelView, "Logs", category="Browse")
+    appbuilder.add_view(views.SlaMissModelView, "SLA Misses", category="Browse")
+    appbuilder.add_view(views.TaskInstanceModelView, "Task Instances", category="Browse")
+    appbuilder.add_view(views.ConfigurationView, "Configurations", category="Admin", category_icon="fa-user")
+    appbuilder.add_view(views.ConnectionModelView, "Connections", category="Admin")
+    appbuilder.add_view(views.PoolModelView, "Pools", category="Admin")
+    appbuilder.add_view(views.VariableModelView, "Variables", category="Admin")
+    appbuilder.add_view(views.XComModelView, "XComs", category="Admin")
+    appbuilder.add_view(views.VersionView, 'Version', category='About', category_icon='fa-th')
+
+
+def init_plugins(app):
+    """Integrate Flask and FAB with plugins"""
+    from airflow import plugins_manager
+
+    plugins_manager.initialize_web_ui_plugins()
+
+    appbuilder = app.appbuilder
+
+    for view in plugins_manager.flask_appbuilder_views:
+        log.debug("Adding view %s", view["name"])
+        appbuilder.add_view(view["view"], view["name"], category=view["category"])
+
+    for menu_link in sorted(plugins_manager.flask_appbuilder_menu_links, key=lambda x: x["name"]):
+        log.debug("Adding menu link %s", menu_link["name"])
+        appbuilder.add_link(
+            menu_link["name"],
+            href=menu_link["href"],
+            category=menu_link["category"],
+            category_icon=menu_link["category_icon"],
+        )
+
+    for blue_print in plugins_manager.flask_blueprints:
+        log.debug("Adding blueprint %s:%s", blue_print["name"], blue_print["blueprint"].import_name)
+        app.register_blueprint(blue_print["blueprint"])
+
+
+def init_error_handlers(app: Flask):
+    """Add custom errors handlers"""
+    from airflow.www import views
+
+    app.register_error_handler(500, views.show_traceback)
+    app.register_error_handler(404, views.circles)
+
+
+def init_api_connexion(app: Flask):
+    """Initialize Stable API"""
+    spec_dir = path.join(ROOT_APP_DIR, 'api_connexion', 'openapi')
+    connexion_app = connexion.App(__name__, specification_dir=spec_dir, skip_error_handlers=True)
+    connexion_app.app = app
+    connexion_app.add_api(
+        specification='v1.yaml', base_path='/api/v1', validate_responses=True, strict_validation=False
+    )
+    app.register_error_handler(ProblemException, connexion_app.common_error_handler)
+
+
+def init_api_experimental(app):
+    """Initialize Experimental API"""
+    from airflow.www.api.experimental import endpoints as e
+
+    # required for testing purposes otherwise the module retains
+    # a link to the default_auth
+    if app.config['TESTING']:
+        import importlib
+
+        importlib.reload(e)
+
+    app.register_blueprint(e.api_experimental, url_prefix='/api/experimental')
+    app.extensions['csrf'].exempt(e.api_experimental)
```

Comment:
It is just used at 2 places so I think the following might be better, WDYT?

```suggestion
    from airflow.www.api.experimental import endpoints

    # required for testing purposes otherwise the module retains
    # a link to the default_auth
    if app.config['TESTING']:
        import importlib

        importlib.reload(endpoints)

    app.register_blueprint(e.api_experimental, url_prefix='/api/experimental')
    app.extensions['csrf'].exempt(endpoints.api_experimental)
```

Reply:
Agree. We don't need an alias here.